homework/style.css

```diff
@@ -1,3 +1,45 @@
 .alert-error {
```

Comment:
Your styling is correct but will not win first prize in a beauty contest, I'm afraid 😉.

![image](https://user-images.githubusercontent.com/2788771/65372858-0767b700-dc76-11e9-9f8b-a127bb944f14.png)